contrib/errorprone/tests/python/pants_test/contrib/errorprone/tasks/test_errorprone_integration.py

```diff
@@ -73,17 +82,24 @@ def test_warning_gets_cached(self):
 
       pants_run = self.run_pants(args)
       self.assert_success(pants_run)
+      self.assertNotIn('Exception caught:', pants_run.stderr_data)
       self.assertIn('[errorprone]', pants_run.stdout_data)
       self.assertIn('No cached artifacts', pants_run.stdout_data)
       self.assertIn('1 warning', pants_run.stdout_data)
+      self.assertIn('SUCCESS', pants_run.stdout_data)
 
       pants_run = self.run_pants(args)
       self.assert_success(pants_run)
+      self.assertNotIn('Exception caught:', pants_run.stderr_data)
       self.assertIn('[errorprone]', pants_run.stdout_data)
       self.assertIn('Using cached artifacts', pants_run.stdout_data)
       self.assertNotIn('No cached artifacts', pants_run.stdout_data)
       self.assertNotIn('1 warning', pants_run.stdout_data)
+      self.assertIn('SUCCESS', pants_run.stdout_data)
 
+  # Unfortunately using the IvyTaskMixin to resolve the javac dependency causes the cache_manager.py
```

Comment:
If it's failing in CI due to a caching issue, it seems very likely to cause you issues in production.

It looks like you're only using `IvyTaskMixin` to get a list of the entries in the classpath... there are other helper methods to do that.